examples/test-app/steps/exits/Done.step.js

```diff
@@ -4,6 +4,14 @@ class Done extends ExitPoint {
   static get path() {
     return '/done';
   }
+
+  values() {
+    return {
+      referenceNumber: 'HDJ2123F',
+      husbandOrWife: this.journey.answers
```

Comment:
Probably better to use `this.journey.values` instead. Returns an object with simple keys you can access. Otherwise we should build the method of "referencing" an value without needing to use a form, something like `this.journey.getField('husbandOrWife', this.journey.steps.RespondentTitle)`.

Reply:
So much better 👍